.gitignore

```diff
@@ -21,6 +21,5 @@ yarn.lock
 *_REMOTE_*
 docs/_site
 docs/_dist
-docs/api
```

Comment:
Why was this removed? Its removal would seem to contradict "README#notes" below

Reply:
Those notes are wrong. I'll update.

The `docs/api` directory was where jsdoc previously placed its output, which jekyll then put into `_site/api`, which was then again built by assetgraph-builder to `docs/_dist`. I updated the jsdoc configuration so `docs/api` is no longer in use